packages/core/strapi/lib/Strapi.js

```diff
@@ -423,7 +413,7 @@ class Strapi {
       }
 
       if (this.config.get('autoReload')) {
-        this.server.destroy();
+        this.destroy();
```

Comment:
It is correct? @alexandrebodin

Reply:
🤔  as the process well be shutdown that makes sense yes.